src/app/forms/templates/test-records/section-templates/vehicle/desk-based-default-hgv-trl-vehicle-section.template.ts

```diff
@@ -0,0 +1,43 @@
+import { ValidatorNames } from '@forms/models/validators.enum';
+import { FormNode, FormNodeEditTypes, FormNodeTypes, FormNodeViewTypes, FormNodeWidth } from '@forms/services/dynamic-form.types';
+import { ReferenceDataResourceType } from '@models/reference-data.model';
+
+export const DeskBasedVehicleSectionDefaultHgvTrl: FormNode = {
+  name: 'vehicleSection',
+  label: 'Vehicle Details',
+  type: FormNodeTypes.GROUP,
+  viewType: FormNodeViewTypes.SUBHEADING,
+  children: [
+    {
+      name: 'vin',
+      label: 'VIN/chassis number',
+      value: '',
+      disabled: true,
```

Comment:
I would like to know the standard approach to this, some devs are hiding these fields others like here are disabling them because the tickets just state not editable

Reply:
Sounds like a PO question to me. Do they want to see it because it comes from the tech record but not edit it or trust that is there (plus they just came from said tech record)?

Reply:
As @gjulien-bjss pointed out, since vin is on the vehicle header component as read-only. In this particular case, hide it.